docs/sources/administration/manage-users-and-permissions/manage-server-users/add-user.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Add a user"
+aliases = ["docs/sources/administration/manage-users-and-permissions/manage-server-users/add-user.md"]
+weight = 10
++++
+
+# Add a user
+
+Add users when you want to provide individuals with access to Grafana.
+
+When you create a user, the system assigns the user viewer permissions in a default organization, which you can change.
+
+<!--- Is there a limit to the number of user created for OSS? -->
+
+<!--- Do orgs need to be created before users? Or does Main Org come out of the box? -->
+
+## Before you begin
+
+- Ensure that you have Grafana server administrator privileges
```

Comment:
Ah, I see you already created http://localhost:3002/docs/grafana/next/administration/manage-users-and-permissions/manage-org-users/invite-user-join-org/. A link from this page to that one would help with that, since inviting a user does actually add a new user unless they already exist in Grafana.

Would still also be nice to add a link out to Cloud user management

Reply:
@mjseaman, unsure of what the ask is here.